data/json/mutations/mutations.json

```diff
@@ -6923,5 +6935,95 @@
     "description": "You have fast reflexes, allowing you to dodge attacks and grabs more easily.",
     "starting_trait": true,
     "dodge_modifier": 2
+  },
+  {
+    "type": "mutation",
+    "id": "THRESH_RABBIT",
+    "name": { "str": "Lagomorph" },
+    "points": 1,
+    "description": "You really want to eat some carrots. Welcome to the Lagomorpha family.",
```

Comment:
```suggestion
    "description": "You really want to eat some carrots.  Welcome to the Lagomorpha family.",
```

Reply:
Thank you, I swear I double checked that, maybe the lint tool messed with it?